website/docs/reference/deploy/environment-import-export.mdx

```diff
@@ -0,0 +1,110 @@
+
+## Import & Export items
+
+When you export features, the export will contain both feature-specific configuration and global configuration.
+
+On the project-level these  items are exported:
+
+* [the feature itself](../feature-toggles.mdx)
+* [feature tags](../tags.md)
+
+On the environment-level, these items are exported for the chosen environment:
+
+* [activation strategies](../activation-strategies.md) including [constraints](../strategy-constraints.md) and references to [segments](../segments.mdx)
+* [variants](../feature-toggle-variants.md)
+* enabled/disabled
+
+Additionally, these global configuration items are exported:
+* [custom context fields](../unleash-context.md#context-fields)
+* [feature tag types](../tags.md#tag-types)
+
+Importantly, while references to [segments](../segments.mdx)) are exported, the segments themselves are **not** exported. Consult the [import rejection](#import-rejection) section for more information.
+
+## Export
+
+You can export features either from a project search or the global feature search. Use the search functionality to narrow the results to the list of features you want to export and use the export button to select environment. All features included in your search results will be included in the export.
+
+<Figure caption="Search-driven export screen with WYSIWYG mode."  img="/img/export.png"/>
+
+## Import
+
+Import is a 3 stage process designed to be efficient and error-resistant.
+
+### Import stages
+
+* **upload** - you can upload previously exported JSON file or copy-paste export data from the exported JSON file into the code editor
+* **validation** - you will get feedback on any errors or warnings before you do the actual import. This makes sure your feature flags configurations. You will not be able to finish the import if you have errors. Warnings don't stop you from importing.
+* **import** - the actual import that creates a new configuration in the target environment or creates a [change request](../change-requests.md) when the environment has change requests enabled
+
+![Three-staged import process](/img/import.png)
+
+### Import rejection
+
+An import will be rejected when there's a possibility of data corruption.
+
+#### Context fields
+
+The exported and imported context fields have the same name but the target environment doesn't support some of the source environment context field values. If the context fields don't exist in the target environment they will be created.
+
+#### Segments
+
+The target Unleash instance doesn't have a matching segment. The actual constraints in the segment can be different. The matching is done by the segment name.
+
+#### Custom strategies
+
+The exported strategies include custom strategies that don't exist in the target Unleash instance. The matching is done by the strategy name.
```

Comment:
```suggestion
If your import contains custom strategies, then custom strategies with the same names must already exist in the target Unleash instance. The custom strategy definitions (including strategy parameters) that exist in the target instance do not otherwise need to match the custom strategy definitions in the instance the import came from.
```